WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -379,6 +385,15 @@ class PostListViewController: AbstractPostListViewController, UIViewControllerRe
         filterSettings.setFilterWithPostStatus(initialFilterWithPostStatus)
     }
 
+    /// Listens for the app coming to foreground in order to properly set the create button
+    private func listenForAppComingToForeground() {
+        NotificationCenter.default.addObserver(forName: UIApplication.willEnterForegroundNotification,
```

Comment:
👋 @Gio2018 - I was looking at [the docs](https://developer.apple.com/documentation/foundation/notificationcenter/1411723-addobserver) for `addObserver(forName:object:queue:using:)` and I wonder if we should be explicitly removing the observer when the view goes away.

The `removeObserver(_:name:object:)` [docs say](https://developer.apple.com/documentation/foundation/notificationcenter/1407263-removeobserver):

> If you used [addObserver(forName:object:queue:using:)](https://developer.apple.com/documentation/foundation/notificationcenter/1411723-addobserver) to create your observer, you should call this method or removeObserver(_:) before the system deallocates any object that addObserver(forName:object:queue:using:) specifies.

> If your app targets iOS 9.0 and later or macOS 10.11 and later, and you used addObserver(_:selector:name:object:) to create your observer, you do not need to unregister the observer. If you forget or are unable to remove the observer, the system cleans up the next time it would have posted to it.

If we do, does the same apply for the [site title PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/18543/files#diff-84a58b7cfc66828e04468ded5b553cec2b9bea10b14d5f92549499ecf4f976d6R63)?

Reply:
Thanks @twstokes !! Good point, we should either call `addObserver(_:selector:name:object:)` or remove the observer. I'll update it. We should indeed do the same in Site Title, but I'd do it in a separate pr since this addresses a specific issue.

Reply:
Sounds good! Apologies for not spotting it on the other PR.